src/main/java/baritone/utils/schematic/litematica/ILitematicaBlockStatePalette.java

```diff
@@ -0,0 +1,27 @@
+package baritone.utils.schematic.litematica;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.nbt.ListNBT;
+
+import javax.annotation.Nullable;
+
+public interface ILitematicaBlockStatePalette
```

Comment:
Also taken from the Minecraft source. PLEASE EXTEND BASE CLASSES.

Reply:
IBlockStatePalette is non-public in minecraft source